starfish/image/_stack.py

```diff
@@ -3,14 +3,15 @@
 from functools import partial
```

Comment:
Four changes here: 
1. Exchange "import numpy" -> "import numpy as np"; this was the only module that has the old import and i thought it was confusing. 
2. move synthetic tile generation here
3. Add a method to generate synthetic ImageStacks from Codebooks
4. add a constructor for ImageStack that consumes a properly shaped numpy array that is already in memory.

Reply:
Extracted by @ttung . Thanks!